app/javascript/gobierto_data/webapp/components/Dataset.vue

```diff
@@ -126,13 +125,12 @@ export default {
       recentQueries: [],
       resourcesList: [],
       currentQuery: null,
-      items: [],
+      items: '',
```

Comment:
Why is items now by default a string? Sounds like this variable should be an empty array by default

Reply:
Because now we receive from the API a string with all the CSV information instead of a JSON array.

Reply:
Hummmm, it's confusing. Even though we'll get an CSV... who is transforming into an array? Are you passing to all children components a string? Wouldn't be better parse the endpoint response in this component, and set `items` as an array for all its children?

Reply:
right, is a CSV, thanks!

Reply:
`Dataset.vue` call API and get response, props-down to `DataTab.vue` > `SQLEditorResults.vue `> `Visualizations.vue`